TableView/Controllers/FakeDataController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  FakeDataController.swift
+//  TableView
+//
+//  Created by Oleg Yankiwskyi on 4/26/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import UIKit
+
+class FakeDataController: UIViewController, FakeDataProtocol, UITableViewDelegate, UITableViewDataSource {
+    var arrayData: Array<CellStatus> = []
```

Comment:
It's a bad practice to include property type into it's name.
If you ever change it to Dictionary or Set, name will confuse your code consumer.